static/src/js/actions/__tests__/focusedCollection.test.js

```diff
@@ -161,8 +161,8 @@ describe('getFocusedCollection', () => {
         expect(getSearchGranulesMock).toHaveBeenCalledTimes(1)
       })
 
-      describe('when the requested collection is cwic and a polygon search is active', () => {
-        test('should toggle the polygon warning, update the focusedCollection and call getSearchGranules', async () => {
+      describe('when the requested collection is cwic and a polygon search is active and we try and retrieve an existing gibs tag', () => {
```

Comment:
"when the requested collection is cwic and a polygon search is active and a GIBS tag exists" maybe?